src/avram/migrator/index_statement_helpers.cr

```diff
@@ -4,13 +4,18 @@ module Avram::Migrator::IndexStatementHelpers
   private getter index_statements = [] of String
 
   # Generates raw sql for adding an index to a table column. Accepts 'unique' and 'using' options.
-  def add_index(column : Symbol, unique = false, using : Symbol = :btree)
+  def add_index(column : Columns, unique = false, using : Symbol = :btree)
```

Comment:
This actually fixes #75

Reply:
ohhh good you pointed the issue, I had this problem and changed (the `Columns` constant was already there heheh) also added the `drop_index` helper

Reply:
Woo hoo!!